### DISCUSSION TITLE
Software bug? EndeavourOS Arch Linux

### DISCUSSION THREAD

**Discussion by rcec52** [Stellarium/stellarium#3149] [General]
I'm having issues with the latest version of Stellarium 23.1 . The graphical landscape display is messed up. I'm using a Nvidia GeForce 1060 video card in an older computer. Previous versions worked fine. All of my software is up to date and I'm using the latest kernel from my OS. I've downloaded and installed files from the AUR arch repository, I also download the appimage from this site and installed the necessary software to launch and install this version on this computer. I still have the same issue with the graphical landscape display. It is messed up. Any idea's what can be causing this? I haven't a clue. 

![Screenshot from 2023-03-31 19-53-37](https://user-images.githubusercontent.com/129566671/229260337-09d1f0c4-f909-4aec-b971-19dfa601d6c0.png)

**Comment by alex-w** [comment]
Looks like a bug - see #3150
